### ISSUE TITLE
Missing a document language on the html tag

### ISSUE THREAD

**Issue by bsoetarto7** [KC74/project_instanews#2]
The `html` tag is missing a document language, `lang="en"`, for accessibility purpose. 

![issue3](https://user-images.githubusercontent.com/6547713/28700695-3e276366-7306-11e7-90e9-22fa2e143735.png)

**Comment by KC74** [comment]
Done
